snippets/elixir.snippets

```diff
@@ -1,81 +1,81 @@
 snippet do
 	do
-		${0}
+		${0:expression}
```

Comment:
A. I missed that. I'll undo _all_ my changes to elixir.snippets changes, then. I thought it was nicer this way, but project standards are a thing. Thanks.